### PR TITLE
Add firefox/retention/thank-you.ftl

### PR DESCRIPTION
**Pull request by alexgibson**
Ref: https://github.com/mozilla/bedrock/pull/9927

### CODE REVIEW

**Comment by alexgibson** [comment]
@peiying2 I think this is the very last migration 🎉

File: en/firefox/retention/thank-you.ftl [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+thank-you-make-firefox-your-default = Make { -brand-name-firefox } your default browser
+thank-you-1-min-action = 1 min action
+thank-you-set-as-your-default = Set as your default
+thank-you-get-firefox-on-your-phone = Get { -brand-name-firefox } on your phone
+thank-you-2-min-install = 2 min install
+thank-you-download-firefox = Download { -brand-name-firefox }
+thank-you-tell-your-friends-about = Tell your friends about { -brand-name-firefox }
+thank-you-1-min-share = 1 min share
+thank-you-join-me-in-the-fight-for = Join me in the fight for an open web by choosing { -brand-name-firefox }!
+thank-you-send-a-tweet = { -brand-name-send } a tweet
```
Comment (peiying2) [inline en/firefox/retention/thank-you.ftl:20]: Send is no longer available.  Let's remove it. And let's remove the entries in the  **brands.ftl** file as well

**Review by peiying2 — CHANGES_REQUESTED** [review]
Other than removing "Send", a product no longer available,  it looks good. Also,  we need to remove the Send entries in the brands.ftl file.

**Comment by alexgibson** [comment]
@peiying2 updated thank you. Note we can't remove a brand name without also updating references to it in every .ftl file for the whole codebase. So rather than do that here I've added Firefox Send to the legacy products grouping.

**Review by peiying2 — COMMENTED** [review]
@alexgibson looks good. I will approve it. 

I do have a question. When strings with the same IDs are moved, would they appear as new strings in Pontoon, or they are magically rearranged in the localized file, thus the communities don't see the changes.

**Comment by alexgibson** [comment]
> I do have a question. When strings with the same IDs are moved, would they appear as new strings in Pontoon, or they are magically rearranged in the localized file, thus the communities don't see the changes.

I'm not 100% sure, but it would appear as a change as far as Git is concerned. I'm not sure how Pontoon interprets that level of change though.

**Comment by peiying2** [comment]
> > I do have a question. When strings with the same IDs are moved, would they appear as new strings in Pontoon, or they are magically rearranged in the localized file, thus the communities don't see the changes.
> 
> I'm not 100% sure, but it would appear as a change as far as Git is concerned. I'm not sure how Pontoon interprets that level of change though.

Just checked with our tech expert, once the strings are translated, the string rearrangement won't trigger it to appear as a new string.

**Review by peiying2 — APPROVED** [review]

**Pull request merged**